core/storage/exploration/gae_models.py

```diff
@@ -349,6 +349,8 @@ class ExpSummaryModel(base_models.BaseModel):
     # The user_ids of users who have contributed (humans who have made a
     # positive (not just a revert) change to the exploration's content)
     contributor_ids = ndb.StringProperty(indexed=True, repeated=True)
+    # A dict representing number of non-trivial commits of user_ids
```

Comment:
A dict representing the contributors of non-trivial commits to this exploration. Each key of this dict is a user_id, and the corresponding value is the number of non-trivial commits that the user has made.